Tasks/NuGetInstallerV0/make.json

```diff
@@ -3,7 +3,8 @@
         {
             "items": [
                 "node_modules/azure-pipelines-tasks-packaging-common/node_modules/azure-pipelines-task-lib",
-                "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib"
+                "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib",
+                "node_modules/azure-pipelines-tool-lib/node_modules/azure-pipelines-task-lib"
```

Comment:
Why is this needed?

Reply:
Some checks failed saying this appeared twice in node_modules, so I removed one of its instances